extensions/kafka/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,9 @@
+=== Quarkus Kafka Dev Services
+
```

Comment:
Do you think there would be advantages to have a single sentence per line ? Not sure how easy it would be for review.

Reply:
Yes, I can change it.